Cargo.toml

```diff
@@ -27,7 +27,7 @@ mime_guess = "2.0.0-alpha.6"
 serde = "1.0"
 serde_json = "1.0"
 serde_urlencoded = "0.5"
-tokio = { version = "0.1.7", default-features = false, features = ["rt-full", "tcp"] }
+tokio = { version = "0.1.7", default-features = false, features = ["rt-full", "tcp", "fs"] }
```

Comment:
Is it possible to only enable this feature as a dev-dependency? We don't otherwise need the extra code *inside* reqwest...

Reply:
Yeah, should be in dev-dependencies of course.

Reply:
Updated Cargo.toml and also added a small refactoring to Body::from<Stream> so it's generic now.